docs/en/logs/logs-installation.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[install-logs-monitoring]]
+[role="xpack"]
+== Setting up logs monitoring
+
+To set up logs monitoring, you need:
+
+* An Elasticsearch cluster and Kibana (version 6.5 or later) with a basic license
```

Comment:
Can we leave out the license part, or describe it as the default distribution?

An Elasticsearch cluster and Kibana (version 6.5 or later).

or 

An Elasticsearch cluster and Kibana (ensure it is Elastic's default distribution, version 6.5 or later).

Reply:
This is a wider issue than this PR. This PR simply relates to the work to split the existing Infrastructure Monitoring Guide "book" into two separate "books" for Metrics and Logs, so that text was already there as written.

Firstly, I'm not sure that this statement is consistently applied in other places where there is a dependency on the license type and/or the version.

Secondly, is it even appropriate to be mentioning older versions of the Elastic stack in the 7.5 documentation? It's quite possible that some of the functionality described for the Logs app in the 7.5 version of the docs won't be available in earlier versions, but by stating an earlier version as a prerequisite, we imply that it will work as described. 

It may be appropriate to mention a version when a feature is first introduced (I think Logs was introduced in 6.5?) but I'm not sure it adds value after that?

Reply:
Yup, agreed, this is a larger issue that should be independent of this PR.